models/user.go

```diff
@@ -28,6 +28,11 @@ type User struct {
 	Free                   bool    `gorm:"type:boolean"`
 	Credits                float64 `gorm:"type:float;default:0"`
 	CustomerObjectHash     string  `gorm:"type:varchar(255)"`
+	// the organization if any this user belongs to
+	// for organization user owners this will be empty.
+	// a non-nil organization field changes how the backend
```

Comment:
The field is not nullable

Reply:
What do you mean? It is allowed to be nullable, since since it is null (empty) that means the user does not belong to an organization, and is then subject to regular billing.